src/AnnotationDialog.js

```diff
@@ -590,14 +565,11 @@ class AnnotationDialog extends EventEmitter {
             case constants.DATA_TYPE_POST:
                 this.postAnnotation();
                 break;
-            // Clicking 'Cancel' button to cancel the annotation
+            // Clicking 'Cancel' button to cancel the annotation OR
+            // Clicking 'X' button on mobile dialog to close
             case constants.DATA_TYPE_CANCEL:
-                if (this.isMobile) {
-                    this.hide();
-                } else {
-                    // Cancels + destroys the annotation thread
-                    this.cancelAnnotation();
-                }
+            case constants.DATA_TYPE_MOBILE_CLOSE:
```

Comment:
Minor, but it seems like some places you are doing constants.something, and in other cases using destructuring for constants

Reply:
We only destructured them when we were using just a few constants but it might make sense to just import the whole thing in a separate commit